examples/pinocchio/src/common.rs

```diff
@@ -0,0 +1,36 @@
+use lambdaworks_math::{
+    elliptic_curve::{
+        short_weierstrass::curves::bls12_381::{
+            curve::BLS12381Curve, default_types::FrElement, default_types::FrField as FrF,
```

Comment:
Given you're aliasing `FrField` later, you may just import with the real name:
```suggestion
            curve::BLS12381Curve, default_types::FrElement, default_types::FrField,
```